index.js

```diff
@@ -1,10 +1,8 @@
 'use strict';
+const {prototype, getPrototypeOf} = Object;
+const {toString} = prototype;
+const prototypes = new Set([null, prototype]);
```

Comment:
Using a Set for just two elements is wasteful. The current check is both faster and simpler.